src/classes/Recipe.js

```diff
@@ -0,0 +1,35 @@
+class Recipe {
+    constructor(recipeDetails) {
+        this.id = recipeDetails.id;
+        this.name = recipeDetails.name;
+        this.image = recipeDetails.image;
+        this.portions = recipeDetails.portions;
```

Comment:
Could you add a default empty array in case portions doesn't get passed in? it could be necessary to prevent errors in other areas of the code that try to reference portions[0] for example
`recipeDetails.portions || []`

Reply:
Sounds good,  this was added to Class and test suite.